File: BackEnd/Planilla/DTO/Planilla/DetallePlanillaDTO.cs
using System;

namespace Planilla.DTO.Planilla
{
    public class DetallePlanillaDTO
    {
        public int DetallePlanillaId { get; set; }
        public int EncabezadoPlanillaId { get; set; }
        public int EmpleadoId { get; set; }
        public decimal Salario { get; set; }
        public decimal DescuentoIsss { get; set; }
        public decimal DescuentoAfp { get; set; }
        public decimal DescuentoRenta { get; set; }
        public decimal OtrosDescuentos { get; set; }
        public decimal? SueldoNeto { get; set; }
        public bool? Activo { get; set; }
        public string CodigoEmpleado { get; set; }
        public DateTime Creado { get; set; }
    }
}

File: FrontEnd/src/app/DTO/DetallePlanillaDTO.ts
export class DetallePlanillaDTO {
    detallePlanillaId:number;
    encabezadoPlanillaId:number;
    empleadoId:number;
    salario:number;
    descuentoIsss:number;
    descuentoAfp:number;
    descuentoRenta:number;
    otrosDescuentos:number;
    sueldoNeto:number;
    codigoEmpleado?: string;
    activo?: boolean;
    actualizar?: boolean;
    creado?: Date;
}